### DISCUSSION TITLE
【深度理财 30】如何办一张汇丰香港卡 - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#19] [Announcements]
# 【深度理财 30】如何办一张汇丰香港卡 - forecho's Blog

引言
受疫情影响香港不知道什么时间解封，而目前在大陆见证办香港卡的渠道是越来越少了。本篇文章就是来分享我如何办汇丰香港卡的经历的。
办卡门槛

费用：0
条件：港澳通行证（签证过期不用管，证件没过期就行），不需要护照
门槛：50 万理财，但是如果是上市公司，在他们企业白名单内，可以不需要 50 万理财
时间：下卡时间要 1 个月左右，但是作为汇丰卓越理财卡，如果卡里面余额不够 50 万的话，会收 300 元人民币/月的服务费。

流程
预约
如果要办，可以加我微信号「ipzone」，暗号「汇丰」，我发你开户预约链接。

https://blog.forecho.com/financedeep-30.html

**Comment by alanhe421** [comment]
除了玩股票，觉得这卡用途还是太少，没信用卡方便。

这张香港卡属于银联，不是Visa，无法直接用卡去玩电商消费。

**Reply by forecho** [reply]
不玩股票办这个卡干啥？主要是为了出入金方便。

**Comment by wpzzz** [comment]
如果大陆卡销户，影响香港卡吗

**Reply by forecho** [reply]
不太清楚，你可以咨询一下客服

**Comment by Jesse1321** [comment]
工银亚洲过期可以在北京广安门工行总行办理，无需赴港。

**Comment by huajialue04** [comment]
请教下，如果从国内卓越理财降级到运筹的话，香港卡那边的功能会受影响吗？

**Reply by forecho** [reply]
香港卡会变成 ONE 卡（给你寄一张新卡），使用起来没什么区别，可能一些权益没有了。

**Reply by huajialue04** [reply]
one卡的出入金受限制吗？和国内运筹互转外汇也不受影响吗？

**Reply by forecho** [reply]
不受限制，卓越理财转钱不收手续费，运筹转到港卡收 54  港币/笔
